activitysim/abm/models/non_mandatory_tour_frequency.py

```diff
@@ -324,6 +331,9 @@ def non_mandatory_tour_frequency(
 
         choices_list.append(choices)
 
+    # FIXME only want to keep actual purposes, adding cols in alts will mess this up
+    # this is complicated by canonical_ids calculated based on alts if not specified explicitly
+    # thus, adding column to input alts will change IDs and break estimation mode....
```

Comment:
I believe this is fixed given the last discussion and code change here -> https://github.com/ActivitySim/activitysim/pull/661

Reply:
Agreed.  Removed outdated comment.